libs/shared/src/consts/providers/credentials/secure-credentials.ts

```diff
@@ -4,6 +4,5 @@ export const secureCredentials: CredentialsKeyEnum[] = [
   CredentialsKeyEnum.ApiKey,
   CredentialsKeyEnum.SecretKey,
   CredentialsKeyEnum.Token,
-  CredentialsKeyEnum.Secure,
```

Comment:
@scopsy Do you know of a possible reason why we would hide/encrypt this variable? (i think it is mistaken to be there in the first place) This is a variable that is boolean and at the moment used only on Nodemailer.

Reply:
I aggree it shouldn't be secure, even tho the name is a bit confusing 😅